ga/src/main/java/nl/tudelft/serg/evosql/fixture/type/DBTypeSelector.java

```diff
@@ -43,21 +43,21 @@ public DBType create(int dataType, int length) {
             case Types.LONGNVARCHAR:
                 return new DBString(EvoSQLConfiguration.MAX_STRING_LENGTH, "LONGNVARCHAR");
             case Types.CHAR:
-                return new DBString(length, "CHAR");
+                return new DBString(length, "CHAR(" + length + ")");
             case Types.NCHAR:
                 return new DBString(length, "NCHAR");
             // Boolean values
             case Types.BIT:
-                return new DBBoolean(); // HSQLDB doesn't support bit
```

Comment:
`ga` does not recognize the `BOOLEAN` type anymore. This should be undone.

Reply:
I haven't tried this out yet, but what happens if you change it to `return new DBBoolean("BIT");`? Would that work?

Reply:
I suppose it would, yes.